src/components/Main/index.jsx

```diff
@@ -2,25 +2,31 @@ import React from 'react';
 import PropTypes from 'prop-types';
```

Comment:
General: 
- Better to not mix styles css/functional css
- Error messages and location display can be separate components as well, what do you think?

Reply:
1. Do you mean have separated file for css?
2. Totally agree. There's room for improvement in that field. I thought about that while doing it. Have more presentational components to ´componentizise´ the solution.

Reply:
1. Yes, because you have a part using constants and another using .css files
2. perfect!

Reply:
1. Ok! And what do you think is the best approach here?
Since I come from native, generally all the styling is defined in the same place with the component there. 
I used a lot this library https://github.com/styled-components/styled-components (11k+ stars).